### ISSUE TITLE
After installing .NET 7 preview everything MAUI related disappears

### ISSUE THREAD

**Issue by tomrus88** [dotnet/maui#6396]
### Description

After installing .NET 7 Preview (preview 3 in my case) all MAUI templates disappear from Visual Studio (17.2.0 preview 4.0) and also all workloads disappear from `dotnet workload list` output.

So this basically makes MAUI completely not usable.

### Steps to Reproduce

Install .NET 7 Preview 3 and everything is gone.

### Version with bug

Release Candidate 1 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

I was *not* able test on other platforms

### Affected platform versions

Windows 10 SDK 10.0.19041.0

**Comment by zzhoney520** [comment]
我的也一样，我认为这是一个很严重的问题，关于maui的负载几乎全部消失，我真不知道这到底是怎么一回事！

**Comment by Fabi** [comment]
Found out, that the only workaround to even create and use (that is documented tho) a maui project is by creating a global.json with the .net 6 framework in the folder you want to create a new maui project. from that folder you can do a dotnet maui then. YOu also need that global.json in the project folder after. Creating projects through VS 2022 is not possible at all with .net 7 installed tho

**Comment by v-longmin** [comment]
Verified repro on windows10.

**Comment by shimingsg** [comment]
I guess it is related to #6181

**Comment by Redth** [comment]
Yes currently you need a global.json to pin to 6.0.x to use MAUI until we support building from .NET 7
